crates/iota-rest-kv/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "iota-rest-kv"
+version.workspace = true
+authors = ["IOTA Foundation <info@iota.org>"]
+edition = "2021"
+license = "Apache-2.0"
+publish = false
+
+[dependencies]
+# external dependencies
+anyhow.workspace = true
+aws-config = "1.5.6"
+aws-sdk-dynamodb = "1.42"
```

Comment:
why are those those crates not part of the root workspace btw?
I can find them in several crates. Maybe @iotaledger/dev-tools knows?

Reply:
I was wondering the same thing, if needed I can make them part of the root workspace, let me know @muXxer @iotaledger/dev-tools